testing/matchers/be_successful_state.go

```diff
@@ -0,0 +1,64 @@
+package matchers
+
+import (
+	"fmt"
+
+	"github.com/onsi/gomega/types"
+	"github.com/pivotal-cf/brokerapi/domain"
+)
+
+const MaxConcurrentFailures = 2
+
+// BeSuccessState is a gomega custom matcher for checking that the result of
+// receiving from a `chan domain.LastOperationState` is a successful state. It
+// mostly works like the `Receive(Equal(state))` matcher, but with the addition
+// that it understands how to bail out early from terminal conditions, making
+// it useful for use in Eventually calls polling the state chan
+func BeSuccessState() types.GomegaMatcher {
```

Comment:
this file is called `be_successful_state.go`, it should be `be_success_state.go` to match the matcher name.